core/Pos/Binary/Class.hs

```diff
@@ -58,7 +58,10 @@ module Pos.Binary.Class
        , biSize
        ) where
 
+import           Universum
```

Comment:
I don't object, but I want to know others' opinions. I don't really care about this particular rule, I only want our code to correspond to style-guide.